website/docs/users-sources/sources/social-logins/google/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Google Identity Providers
+sidebar_label: Google
+tags: [integration, saml, google]
+---
+
+Google provides several services that can be used as identity providers in authentik.
+This page provides an overview of how to configure each service.
+
+## Google Workspace (SAML)
```

Comment:
Let's match the same order shown in the left nav pane... so have Cloud first, then GWS...